superset-frontend/src/dashboard/components/DashboardBuilder/DashboardBuilder.tsx

```diff
@@ -61,12 +61,19 @@ const HEADER_HEIGHT = 67;
 
 type DashboardBuilderProps = {};
 
-const StyledDashboardContent = styled.div<{ dashboardFiltersOpen: boolean }>`
+const StyledDashboardContent = styled.div<{
+  dashboardFiltersOpen: boolean;
+  editMode: boolean;
+}>`
   display: flex;
   flex-direction: row;
   flex-wrap: nowrap;
   height: auto;
   flex-grow: 1;
+  padding-left: ${({ theme, editMode }) => {
```

Comment:
Just a nit. Maybe just go straight to return for the sake of brevity

Reply:
I did but then it raises https://eslint.org/docs/rules/no-confusing-arrow. That's why it's using curly braces.